app/components/enterprise-banner.js

```diff
@@ -0,0 +1,122 @@
+import Ember from 'ember';
+
+import { computed } from 'ember-decorators/object';
+import { alias } from 'ember-decorators/object/computed';
+import { service } from 'ember-decorators/service';
+
+import timeAgoInWords from 'travis/utils/time-ago-in-words';
+
+export default Ember.Component.extend({
+  @service ajax: null,
+  @service storage: null,
+
+  key: 'travis.enterprise.license_msg_last_seen',
+
+  didInsertElement() {
+    this._super(...arguments);
+
+    const url = '/v3/enterprise_license';
+    Ember.run(() => {
+      this.get('ajax').get(url).then(response => {
+        const exp = new Date(Date.parse(response.expiration_time));
+        this.set('licenseId', response.license_id);
+        this.set('expirationTime', exp);
+        this.set('daysUntilExpiry', Math.ceil(
+          (exp.getTime() - new Date().getTime()) / (1000 * 60 * 60 * 24)
+        ));
+        this.set('licenseType', response.license_type);
+        this.set('seats', response.seats);
+        this.set('activeUsers', response.active_users);
+        this.set('isTrial', response.license_type === 'trial');
+        this.set('isPaid', response.license_type !== 'trial');
```

Comment:
This could be a good place for [`setProperties`](https://www.emberjs.com/api/ember/2.18/classes/EmberObject/methods/setProperties?anchor=setProperties) 😀

Reply:
Nice, didn't know about this!